02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -333,15 +336,169 @@ Looks like we were able to successfully get rid of the duplicate gene identifier
 
 ## Perform gene set enrichment analysis (GSEA)
 
-_Addressed in upcoming PR_
+The goal of GSEA is to determine whether the members of a set of genes are randomly distributed throughout a pre-ranked gene list or primarily found at the top or bottom of the list meaning that they are highly- or lowly-ranked.
```

Comment:
Is this the goal of the analysis or is it a description of the approach we use in service of our goal?

Reply:
Good question, this goal was inspired by the goal in the related GSEA article: https://www.pnas.org/content/102/43/15545 which states that
"The goal of GSEA is to determine whether members of a gene set S tend to occur toward the top (or bottom) of the list L, in which case the gene set is correlated with the phenotypic class distinction."

However, now that you point it out, I would classify this as a description of the approach we use in service of our goal. Will update appropriately 👍